modules/javafx.graphics/src/test/java/test/com/sun/javafx/css/parser/CssLexerTest.java

```diff
@@ -45,7 +44,7 @@ public class CssLexerTest {
     public CssLexerTest() {
     }
 
-    private void checkTokens(List<TokenShim> resultTokens, TokenShim... expectedTokens) {
+    private void checkTokens(List<Token> resultTokens, Token... expectedTokens) {
```

Comment:
should there be new tests in `CssLexerTest` for the `@media` queries?

Reply:
No, media queries are not a thing at the lexer stage. The media query grammar only comes in at a later stage.

Reply:
`CssLexer` is responsible for converting the stylesheet text into a token stream. In this stream, `@media` appears as an `AT_KEYWORD` token, but no further syntactic processing is done (i.e. the parser doesn't care at all if the sequence of tokens makes any sense). This only happens in `CssParser`, where the actual value of the token is inspected and subsequent tokens are interpreted according to a specific grammar (in this case, the media query grammar). That's why all of the parsing tests are in `CssParser_mediaQuery_Test`.